kms/src/test/java/com/example/SnippetsIT.java

```diff
@@ -204,6 +204,24 @@ public void destroyCryptoKeyVersion_destroys() throws Exception {
         KEY_RING_ID, CRYPTO_KEY_ID, version));
   }
 
+
+  @Test
+  public void restoreCryptoKeyVersion_restores() throws Exception {
+    Snippets.createCryptoKeyVersion(PROJECT_ID, LOCATION_ID, KEY_RING_ID, CRYPTO_KEY_ID);
+
+    Matcher matcher = Pattern.compile(".*cryptoKeyVersions/(\\d+)\",\"state\":\"DESTROY_SCHEDULED\".*",
```

Comment:
This might be a bit brittle.  What if it's actually done?

Reply:
Do you mean if the destruction already occurred after the wait period?

Reply:
Yes